src/components/flex/flex_item.tsx

```diff
@@ -46,13 +46,48 @@ export type FlexItemGrowSize =
   | true
   | false
   | null;
+export type FlexItemShrinkSize =
+  | 1
+  | 2
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10
+  | true
+  | false
+  | null;
```

Comment:
We should add `0` to both shrink and grow. The real CSS range for both is `<number [0,∞]>`.

Let's also add ToDo comments to remove true and false values from this in the next major release.

Reply:
@AMoo-Miki,

This is OK?

```js
// TODO: Remove 'true' and 'false' from the type definitions on next major release
export type FlexItemGrowSize =
  | 0
  | 1
  | 2
  | 3
  | 4
  | 5
  | 6
  | 7
  | 8
  | 9
  | 10
  /**
   * @deprecated `true` will be removed in the next major release.
   * Use numeric values instead.
   */
  | true
  /**
   * @deprecated `false` will be removed in the next major release.
   * Use numeric values instead.
   */
  | false
  | null;
  ```
  
  I just pushed these suggested changes for both types `FlexItemGrowSize` and `FlexItemShrinkSize`